master/static/srv/get_trials_for_experiment.sql

```diff
@@ -0,0 +1,16 @@
+SELECT t.id AS id,
+    t.start_time AS start_time,
+    t.end_time AS end_time,
+    t.experiment_id AS experiment_id,
+    t.hparams AS hparams,
+    'STATE_' || t.state AS state,
+    (
+        SELECT s.prior_batches_processed + s.num_batches
+        FROM steps s
+        WHERE s.trial_id = t.id
+            AND s.state = 'COMPLETED'
+        ORDER BY s.id DESC
+        LIMIT 1
+    ) AS batches_processed
```

Comment:
I think this is called `total_batches_processed` in other places, so can we call it that here?